cfpb/cfgov-refresh
# Mega Menu: Add variations styles and JavaScript

## Changes

- Adds styles and JavaScript for mega menu variations.

## Testing

1. Pull branch and run `gulp clean && gulp build`
2. Enable the `MEGA_MENU_VAR_1` flag and reload the homepage and see that the menu is the variation 1 with a caret by the menu items.
3. Enable the `MEGA_MENU_VAR_2` flag (and disable `MEGA_MENU_VAR_1`) and reload the homepage and see that the menu is the variation 2 without a caret by the menu items.

## Note
- This duplicates the MegaMenu.js code, which inflates the common.js bundle a bit. When we finalize the menu changes this will drop back down.
